landingcorp/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "landingcorp",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "compile:sass": "node-sass main.scss css/style.css -w",
+    "devserver": "live-server"
+  },
+  "dependencies": {
+    "live-server": "^1.2.1",
+    "node-sass": "^4.14.0"
+  }
+}
```

Comment:
I'm adding some helper scripts to the package.json in the project root directory. You can simply use them instead of adding package.json files to the template directory. This should be much cleaner :wink: